### DISCUSSION TITLE
Possible bug, Could be my config

### DISCUSSION THREAD

**Discussion by N1GHTR4NG3R** [prisma/prisma#7355] [Q&A] [answered]
I have Prisma set up to connect through an SSH Tunnel using NodeSSH - Here is my configuration:

```
ssh
  .connect({
    host: "IP",
    port: "PORT",
    username: "dbteam",
    privateKey: "./resources/countyLines/Server/Configs/SSH-Key/dbteam.ppk",
    tryKeyboard: true,
  })
  .then(() => {
    console.log("Connected through SSH, attempting Database connection...");
    if (db) {
      console.log("Database connected successfully!");
    } else {
      console.log("Failed to connect to database...");
    }
  });
  ```
  When starting my server this connects successfully, However if I try to interact with the database it will error with (Terminal Log to show connected - and also failure when trying too update):
  
```
Connected through SSH, attempting Database connection...
Database connected successfully!
3 commands loaded!
Name: Nightranger just connected!
[07:02:04][Warning] Event handler at countyLines:file:///A:/Backups/ricky/CountyLines%20RP/countyLines/resources/countyLines/Server/Webviews/Login/login.js:12 was too long 8ms
(node:29772) UnhandledPromiseRejectionWarning: Error: 
Invalid `prisma.account.findMany()` invocation:


  Can't reach database server at `localhost`:`PORT`

Please make sure your database server is running at `localhost`:`PORT`.
    at cb (A:\Backups\ricky\CountyLines RP\countyLines\node_modules\@prisma\client\runtime\index.js:79166:17)
    at emitWarning (internal/process/promises.js:120:15)
    at processPromiseRejections (internal/process/promises.js:168:7)
    at processTicksAndRejections (internal/process/task_queues.js:90:32)
    
 ```
 
 
    I can't understand or see why this is failing, as I have two other applications accessing and updating the database 
    fine with no issues, but using Prisma it fails...

**Comment by N1GHTR4NG3R** [comment]
An update on this question.

I am connecting through Datagrip database GUI with the same details and am able to query and update the database using the same details.

So my configuration appears to be correct - which leads to the the "Invalid `prisma.account.findMany()` invocation - I'll update this to a bug report, and see what I get back.

**Comment by janpio** [comment, marked as answer]
Let's continue this over at https://github.com/prisma/prisma/issues/7356

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
